scripts/explain_manifest/fixtures/ubuntu-24.04-arm64.txt

```diff
@@ -190,7 +190,7 @@
   - lua-resty-lmdb
   - ngx_brotli
   - ngx_wasmx_module
-  OpenSSL   : OpenSSL 3.2.1 30 Jan 2024
+  OpenSSL   : OpenSSL 3.2.3 3 Sep 2024
```

Comment:
Why upgrade the toolchain can change the OpenSSL version?

Reply:
@ADD-SP I think there was some conflict with addition of new Ubuntu version, and some PRs did not see the addition of new Ubuntu. Depending on which was merged first.